.github/workflows/interop-tests.yml

```diff
@@ -0,0 +1,34 @@
+name: Interoperability tests with GnuTLS and NSS
+on: [pull_request, push]
+jobs:
+  test:
+    runs-on: ubuntu-22.04
+    container:
+      image: docker.io/nhorman/fedora38-interop:latest
+      options: --sysctl net.ipv6.conf.lo.disable_ipv6=0
+      volumes:
+        - /home/runner/work/openssl/openssl:/openssl
```

Comment:
This might be not needed as well. `actions/checkout@v4` is executed within the container.

Reply:
yup, you're right, changing to the container syntax you referenced lets us do the checkout inside the container, so volume mounts aren't needed